packages/quantic/force-app/main/default/lwc/quanticCategoryFacetValue/quanticCategoryFacetValue.html

```diff
@@ -39,7 +40,7 @@
               <span class="facet__number-of-results slds-var-m-left_xx-small">({item.numberOfResults})</span>
             </div>
             <div class="slds-col slds-p-left_none">
-              <span class="facet__path">{labels.inLabel} {item.localizedPath}</span>
+              <span class="facet__path slds-truncate">{labels.inLabel} {item.localizedPath}</span>
```

Comment:
don't we need to specify a `title` attribute here as well?

Reply:
Yeah might as well